### PR TITLE
update pupil kernel function to default to unit area + better docs

### PR DESCRIPTION
**Pull request by drammock**
This change follows discussions with @Eric89GXL about sensible values for scaling the pupil kernel.  Should be pretty straightforward; ready for review/merge.

### CODE REVIEW

File: pyeparse/utils.py [hunk 1]
```diff
@@ -46,12 +46,29 @@ def fwhm_kernel_2d(size, fwhm, center=None):
...
+        Desired value for the area under the kernel. If `None`, no scaling is
+        performed.
+    """
+
     n_samp = int(np.round(fs * dur))
     t = np.arange(n_samp, dtype=float) / fs
     h = (t ** n) * np.exp(- n * t / t_max)
-    h = 0.015 * h / (np.sum(h) * (t[1] - t[0]))
+    denom = 1. if s is None else np.sum(h) * (t[1] - t[0])
+    h = s * h / denom
```
Comment (Eric89GXL) [inline pyeparse/utils.py]: this is going to bomb if `s is None`

Comment (drammock) [inline pyeparse/utils.py]: Oops. That's what I get for coding while attending a lecture. Will fix.

**Comment by drammock** [comment]
I think all stupidity has now been removed, @Eric89GXL.

**Comment by Eric89GXL** [comment]
Thanks @drammock

**Pull request merged**